src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltokentext/InputIllegalTokenTextDefaultFormat.java

```diff
@@ -0,0 +1,19 @@
+/*
+IllegalTokenText
+format = (default)^$
+ignoreCase = (default)false
+message = (default)
+tokens = STRING_LITERAL
+
+
+*/
+
+
+package com.puppycrawl.tools.checkstyle.checks.coding.illegaltokentext;
+
+public class InputIllegalTokenTextDefaultFormat {
+    public void myTest() {
+        String test = "a href"; // ok
```

Comment:
Please remove all ok comments and never use them,  it was our mistake to start to use them.

Reply:
If i am not wrong the ant task will be failed in mvn clean verify

Reply:
![image](https://github.com/checkstyle/checkstyle/assets/97679350/13155fa6-be31-499f-8fee-7b3e07d08643)

Reply:
Share it as text

Reply:
Ok, put one unrelated comment over class. To avoid dealing with this Check, I will try to disable it in separate PR.

Reply:
```
[INFO] --- maven-antrun-plugin:3.1.0:run (ant-phase-verify) @ checkstyle ---
[INFO] Executing tasks
[INFO]      [echo] Checkstyle started (checkstyle-checks.xml): 12/06/2023 10:32:21 am
[INFO] [checkstyle] Running Checkstyle  on 1507 files
[INFO]      [echo] Checkstyle finished (checkstyle-checks.xml) : 12/06/2023 10:33:26 am
[INFO]      [echo] Checkstyle started (checkstyle-non-main-files-checks.xml): 12/06/2023 10:32:21 am
[INFO] [checkstyle] Running Checkstyle  on 3657 files
[INFO]      [echo] Checkstyle finished (checkstyle-non-main-files-checks.xml): 12/06/2023 10:33:26 am
[INFO]      [echo]     
[INFO]      [echo] Checkstyle started (checkstyle-resources-checks.xml): 12/06/2023 10:32:21 am
[INFO] [checkstyle] Running Checkstyle  on 3307 files
[INFO]      [echo] Checkstyle finished (checkstyle-resources-checks.xml): 12/06/2023 10:33:26 am
[INFO]      [echo] Checkstyle started (checkstyle-input-checks.xml): 12/06/2023 10:32:21 am
[INFO] [checkstyle] Running Checkstyle  on 2102 files
[ERROR] [checkstyle] [ERROR] /home/kevin/Desktop/check_style/checkstyle/checkstyle/src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltokentext/InputIllegalTokenTextDefaultFormat.java:0: All check input files in the 'src/(it|test)/resources*/' folder must                    have trailing comments "// ok" and/or "// violation". [StyleValidationCommentInInputs]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  03:16 min
[INFO] Finished at: 2023-06-12T10:35:27+05:30
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-antrun-plugin:3.1.0:run (ant-phase-verify) on project checkstyle: An Ant BuildException has occured: The following error occurred while executing this line:
[ERROR] /home/kevin/Desktop/check_style/checkstyle/checkstyle/config/ant-phase-verify.xml:192: Got 1 errors and 0 warnings.
[ERROR] around Ant part ...<ant antfile="config/ant-phase-verify.xml" />... @ 6:50 in /home/kevin/Desktop/check_style/checkstyle/checkstyle/target/antrun/build-main.xml
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.

```

Reply:
@Kevin222004 Simply remove trailing `//ok` comments from variable defs and place one at class level i.e.
```java
public class InputIllegalTokenTextDefaultFormat { // ok
```

Reply:
Done

Reply:
Request to remove ok comments - https://github.com/checkstyle/checkstyle/issues/13213